### PR TITLE
Add project.md to table of contents

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

**Comment by vbatts** [comment]
I had left that out for no particular reason. And it's also not in the
makefile for being included in the docs.

**Comment by hqhq** [comment]
Updated, added it to Makefile as well.

File: Makefile [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-	README.md \
+	bundle.md \
 	code-of-conduct.md \
+	config-linux.md \
+	config.md \
+	glossary.md \
+	implementations.md \
 	principles.md \
-	style.md \
+	project.md \
```
Comment (vbatts) [inline Makefile]: I should add a comment, but these files were in an order that determines how they show up in the PDF/HTML docs. 
If you could just put the project.md entry where it corresponds to where you put it in the table of contents. Thanks.

**Comment by hqhq** [comment]
@vbatts Updated, and added the comment, thanks.

**Comment by vbatts** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**